docs/api.md

```diff
@@ -284,9 +284,12 @@ Create a network, similar to the `docker network create` command.
 **Params**:
 
 * name (str): Name of the network
-* driver (str): Name of the driver used to create the network
-
+* driver (str): Name of the network driver plugin to use. Defaults to bridge driver
```

Comment:
This is incorrect - it defaults to `bridge` for regular Docker Engine, but the default for Swarm is `overlay`. Please remove the second part.